docs/verbs/catkin_build.rst

```diff
@@ -533,6 +533,32 @@ So if you want to check for failing tests, use the ``catkin_test_results`` comma
 
 The result code will be non-zero unless all tests passed.
 
+
+Running Linters Built in a Workspace
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In ROS community, there is a standard `linter <https://github.com/ros/roslint>`_ for C++ and Python.
+To run linters for all catkin packages in the workspace, use the following:
+
+.. code-block:: bash
+
+    $ catkin roslint
+
+Or the longer version:
+
+.. code-block:: bash
+
+    $ catkin build [...] --make-args roslint
```

Comment:
Why not `--catkin-make-args`? There shouldn't be any non-catkin packages with the `roslint` target right?